packages/react-native-gesture-handler/src/v3/hooks/useGesture.ts

```diff
@@ -46,6 +49,69 @@ function shouldHandleTouchEvents(config: Record<string, unknown>) {
   );
 }
 
+const sharedValues = new Map<
+  string,
+  { id: number; sharedValue: SharedValue }
+>();
+
+let nextSharedValueListenerID = 0;
+
+// This is used to obtain HostFunction that can be executed on the UI thread
+const { updateGestureHandlerConfig } = RNGestureHandlerModule;
+
+function maybeExtractSharedValues(config: any, tag: number) {
```

Comment:
```suggestion
function bindSharedValues(config: any, tag: number) {
```

Imo, we don't need `maybe` prefix here. If there's no Reanimated, there will be no shared values - every shared value would be bound. Same, if there's Reanimated but no shared values in the config. And if there's Reanimated and shared values, they will be bound correctly.

Reply:
Sure, done in [fd6eb79](https://github.com/software-mansion/react-native-gesture-handler/pull/3658/commits/fd6eb79354defaff8a91f5ec0f19bd3008ec0ff2)